ldk/javascript/examples/react-loop/src/index.js

```diff
@@ -0,0 +1,576 @@
+import * as React from 'react';
+import '@oliveai/ldk';
+import * as ReactWhisper from '@oliveai/ldk/dist/whisper/react/renderer';
```

Comment:
We're actually bundling a specific version of React with the LDK now and importing React from that, could you give that a shot?

```suggestion
import { React, ReactWhisper } from '@oliveai/ldk';
```